src/main/drivers/pwm_output.c

```diff
@@ -460,8 +456,8 @@ void servoDevInit(const servoDevConfig_t *servoConfig)
             /* flag failure and disable ability to arm */
             break;
         }
-
-        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_MHZ, 1000000 / servoConfig->servoPwmRate, servoConfig->servoCenterPulse, 0);
+        const uint16_t period = PWM_TIMER_HZ / servoConfig->servoPwmRate;
+        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_HZ, period, (servoConfig->servoCenterPulse - 1000) / 1000 * period, 0);
```

Comment:
have reverted back to original code - before the review. The timer is prescaled to ensure 1us per tick.